Caddyfile

```diff
@@ -1,16 +1,11 @@
+{
+	acme_ca https://acme-v02.api.letsencrypt.org/directory
+}
+
 testnet-node-1.cere.network:9945 {
-	log stdout
-	errors stderr
-	gzip
```

Comment:
@groschovskiy 
Don't we need gzip more?

Reply:
IDK